packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -5,6 +5,8 @@ import * as BrowserstackLocalLauncher from 'browserstack-local'
 import logger from '@wdio/logger'
 import type { Capabilities, Services, Options } from '@wdio/types'
 
+// @ts-ignore
```

Comment:
Why is this needed?

Reply:
Was seeing this error: `'rootDir' is expected to contain all source files. error` https://github.com/webdriverio/webdriverio/pull/8350#discussion_r875477783

Reply:
Gotcha, yeah we can ignore this for now